src/Symfony/Component/WebLink/composer.json

```diff
@@ -26,7 +26,7 @@
         "symfony/http-kernel": ""
     },
     "require-dev": {
-        "symfony/http-foundation": "^4.4|^5.0",
+        "symfony/http-foundation": "^5.3",
```

Comment:
conflict should also be updated I think

Reply:
or not :) nvm

Reply:
You are correct. 

That is because `symfony/http-foundation` is only a dev-dependecy.